src/components/SplitVertical/SplitVertical.less

```diff
@@ -0,0 +1,47 @@
+@SplitVertical-transition-duration: @timing-animation-fade;
+@SplitVertical-transition-half-duration: @SplitVertical-transition-duration / 2;
+
+.lucid-SplitVertical {
+	overflow: hidden;
+
+	& > &-inner {
```

Comment:
yes, nested frames break without the direct descendent selectors, and i think it's well formed css since it's styling a component with a very specific DOM structure.